### PR TITLE
Release/v0.4

### PR DESCRIPTION
**Pull request by altendky**

### CODE REVIEW

**Comment by altendky** [comment]
If reviewing, note that there is a changelog to read.

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,50 @@ All notable changes to this project will be documented in this file.
...
+- Identify more cases of chia plotting processes such as on NixOS.
+- Avoid some more `NoSuchProcess` and `AccessDenied` errors when identifying plotting processes.
+- Avoid crashing when parsing plotting process logs fails to decode due to `UnicodeDecodeError`.
+- Avoid crashing when a tmp file is removed while we are checking a job's tmp usage.
+- Windows is not yet supported, but plot and archive processes are now launched to be independent of the plotman process on Windows as it already was on Linux.
+### Added
+- Configuration file is versioned.
+  The config for previous plotman versions has been retroactively defined to be version 0
+  The new version is 1.
+  An error will be raised when launching plotman which includes a link to the wiki.
```
Comment (ericaltendorf) [inline CHANGELOG.md:23]: Something missing here?  Do you mean if an error is raised when launching plotman, a wiki link will be included?

**Review by ericaltendorf — CHANGES_REQUESTED** [review]
Just one question on the release notes.

File: CHANGELOG.md (hunk 1 above)
Comment (altendky) [reply CHANGELOG.md:23]: I reworded it and mucked it up in the process.  Thanks.

```suggestion
  An error will be raised when you launch plotman with a configuration file whose version does not match the expected configuration version.
  That error will include a link to the wiki to help understand the needed changes.
```

**Review by ericaltendorf — APPROVED** [review]

**Pull request merged**